src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeItem.java

```diff
@@ -1,36 +1,52 @@
 package net.tiagofar78.prisonescape.game;
 
+import net.tiagofar78.prisonescape.managers.ConfigManager;
+
 public enum PrisonEscapeItem {
 	
 	// TODO: for now all illegals are set to false
-	// also need to re-check metal items
-    SEARCH(false, false),
-    HANDCUFS(true, false),
-    SPOON(false, false),
-    SHOVEL(false, false),
-    MATCHES(false, false),
-    BOLTS(true, false),
-    DUCTTAPE(false, false),
-    WIRE(true, false),
-    PLASTICPLATE(false, false),
-    OIL(false, false),
-    STICK(false, false),
-    COFFEE(false, false),
-    ENERGYDRINK(false, false);
+	// also need to re-check metal items and rare items
+    SEARCH(false, false, false),
+    HANDCUFS(true, false, false),
+    SPOON(false, false, false),
+    SHOVEL(false, false, false),
+    MATCHES(false, false, false),
+    BOLTS(true, false, false),
+    DUCTTAPE(false, false, false),
+    WIRE(true, false, false),
+    PLASTICPLATE(false, false, false),
+    OIL(false, false, false),
+    STICK(false, false, false),
+    COFFEE(false, false, false),
+    ENERGYDRINK(false, false, false);
 
-    private final boolean isMetal;
-    private final boolean isIllegal;
+    private final boolean _isMetal;
+    private final boolean _isIllegal;
+    private final boolean _isRare;
     
-	PrisonEscapeItem(boolean isMetal, boolean isIllegal) {
-        this.isMetal = isMetal;
-        this.isIllegal = isIllegal;
+	PrisonEscapeItem(boolean isMetal, boolean isIllegal, boolean isRare) {
```

Comment:
Maybe it's better if rarity is a double. For now it's ok but in the future we may want to change it.